src/test/rustdoc/decl_macro.rs

```diff
@@ -13,3 +13,20 @@ pub macro my_macro() {
 pub macro my_macro_2($($tok:tt)*) {
 
 }
+
+// @has decl_macro/macro.my_macro_multi.html //pre 'pub macro my_macro_multi {'
+// @has - //pre '(_) => { ... },'
+// @has - //pre '($foo:ident . $bar:expr) => { ... },'
+// @has - //pre '($($foo:literal),+) => { ... }'
+// @has - //pre '}'
+pub macro my_macro_multi {
```

Comment:
Can you test what this looks like when there's only one matcher, but it uses macro-by-example syntax? I have a feeling it will look kind of odd and you'll end up needing to do the same thing for both the 1 and more than 1 cases.

Reply:
Do you mean adding a test for this?
```rs
pub macro foo {
    ($foo:bar) => {}
}
```
If so: I'll add the test, as for your second point, I don't think it hurts in that specific case to display it as `pub macro foo($foo:bar)`, as long as it looks right, and if we want to make it always display exactly as in source for that case I'd kind of prefer to open an issue then fix that in a follow-up.

Reply:
Yes, that's what I meant. I'm fine with showing that as `pub macro foo($foo: bar)`, I'm just worried it will look like something different because you use `span.to_src`.

Reply:
Ahh, alright. It seems to work fine, the `()` are included in both cases, so they're already not in the format string.